### PR TITLE
UI dashboard migrating Vue js

### LINKED ISSUE #15: (UI) Dashboard html page to VueJs migration
**Description:**

Convert the mock html page to VueJs component of dashboard with hard coded values as it is.

### PR DESCRIPTION
**Pull request by JaswinderSinghdigwa**
migrating dashboard Page html to vue js

### CODE REVIEW

File: FrontEnd/src/views/DashboardView.vue [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+          <div class="card-header">
+            <h4 class="my-0 fw-normal">472 Aanganwadis</h4>
+          </div>
+          <div class="card-body">
+            <h3 class="card-title pricing-card-title">2,306 <small class="text-muted fw-light">[SAM] <i class="bi bi-arrow-up-circle text-danger"></i> 37 </small></h3>
+            <h3 class="card-title pricing-card-title">7,812 <small class="text-muted fw-light">[MAM] <i class="bi bi-arrow-down-circle text-success"></i> 123 </small></h3>
+          </div>
+        </div>
+
+		<div class="card rounded-3 shadow-sm" style="min-height:220px">
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/DashboardView.vue]: Could you please address all the alignment issues in this files and extra lines where ever not required can be removed.

**Review by mjayakumarmca — CHANGES_REQUESTED** [review]
Just one comment

File: FrontEnd/src/views/DashboardView.vue (hunk 1 above)
Comment (JaswinderSinghdigwa) [reply FrontEnd/src/views/DashboardView.vue]: changes updated
Map is not implemented properly please check this issues

**Review by mjayakumarmca — APPROVED** [review]
LGTM

**Pull request merged**